src/model/database.cpp

```diff
@@ -228,6 +228,19 @@ int ObjectDatabase::CountChildrenWithSameType(PObjectBase parent,PObjectType typ
 	return count;
 }
 
+int ObjectDatabase::CountChildrenWithTypes(PObjectBase parent,const std::set<wxString> &types)
```

Comment:
Missing whitespace in front of the comma, attach the reference to the type please